wiki/en/en-Onboarding.md

```diff
@@ -62,6 +62,6 @@ If you need help or want to contribute to the Jamulus Project, you can have a lo
 For help, chat, and general discussions about Jamulus: Have a look at the [forums on SourceForge](https://sourceforge.net/p/llcon/discussion/).
 
 ### Development and contribution
-If you want to contribute to the Jamulus Software, have a look at the [Jamulus GitHub repository](https://github.com/jamuluswebsite/jamulus/) and read the contributing file there.
+If you want to contribute to the Jamulus Software, have a look at the [Jamulus GitHub repository](https://github.com/jamulussoftware/jamulus/) and read the `CONTRIBUTING.MD` file there.
```

Comment:
CONTRIBUTING.MD or CONTRIBUTING.md?

Reply:
Lower case .md. I'd left caps on but corrected it.